crypto/fipsmodule/cpucap/cpucap.c

```diff
@@ -68,6 +68,9 @@ HIDDEN uint32_t OPENSSL_armcap_P =
 #endif
 #if defined(__ARM_FEATURE_SHA512)
     ARMV8_SHA512 |
+#endif
+#if defined(__ARM_FEATURE_SHA3)
```

Comment:
__ARM_FEATURE_SHA3 will only be defined if the processor is native or the -march= specifies it. Don't you want runtime detection?

Reply:
Thank you AGSaidi for taking a look. This section is enclosed within `#if defined(OPENSSL_STATIC_ARMCAP)`. The runtime check is in [cpu_aarch64_linux.c](https://github.com/awslabs/aws-lc/pull/721/files#diff-fcf179e7b494018fd1cfafa8a0372a75f4852211cd483959efb7efa60595c804R37-R42).